test/leafnode_test.go

```diff
@@ -1979,3 +2023,56 @@ func TestLeafNodeSendsRemoteSubsOnConnect(t *testing.T) {
 	_, leafExpect := setupConn(t, lc)
 	leafExpect(lsubRe)
 }
+
+func TestLeafNodeServiceImportLikeNGS(t *testing.T) {
+	server.SetGatewaysSolicitDelay(10 * time.Millisecond)
+	defer server.ResetGatewaysSolicitDelay()
+
+	ca := createClusterEx(t, true, "A", 3)
+	defer shutdownCluster(ca)
+	cb := createClusterEx(t, true, "B", 3, ca)
+	defer shutdownCluster(cb)
+
+	// Hang a responder off of cluster A.
+	opts := ca.opts[0]
+	url := fmt.Sprintf("nats://ngs:pass@%s:%d", opts.Host, opts.Port)
+	nc, err := nats.Connect(url)
+	if err != nil {
```

Comment:
I have created helpers in the past to avoid having to manually check. For instance, you could use:
```
nc := natsConnect(t, url)
```

Reply:
These in test dir or under server?

Reply:
Not available under test.

Reply:
Yes, sorry, they are under `server/`. In the future we could duplicate those in `test/` too since they offer error checking but allow test code to be streamlined.